uint32 type
string vehicleName
float64 x
float64 y
float64 z
float64 vx
float64 vy
float64 vz
float64 roll
float64 pitch
float64 yaw
float64 rollRate
float64 pitchRate
float64 yawRate
float64 acquiringTime #delta t
bool isValid